ng-dev/pr/merge/pull-request.ts

```diff
@@ -85,15 +85,17 @@ export async function loadAndValidatePullRequest(
     throw error;
   }
 
-  /** List of parsed commits for all of the commits in the pull request. */
-  const commitsInPr = prData.commits.nodes.map((n) => parseCommitMessage(n.commit.message));
+  if (config.labels !== false) {
+    /** List of parsed commits for all of the commits in the pull request. */
+    const commitsInPr = prData.commits.nodes.map((n) => parseCommitMessage(n.commit.message));
 
-  try {
-    assertPendingState(prData);
-    assertChangesAllowForTargetLabel(commitsInPr, targetLabel, config);
-    assertCorrectBreakingChangeLabeling(commitsInPr, labels, config);
-  } catch (error) {
-    return error;
+    try {
+      assertPendingState(prData);
```

Comment:
I feel like the pending state and correct breaking change stuff is unrelated to the labeö
config and should still run?

Reply:
Good call.